modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -518,10 +518,10 @@ PropertyInfo VisualScriptBuiltinFunc::get_output_value_port_info(int p_idx) cons
 			t = Variant::REAL;
 		} break;
 		case MATH_ABS: {
-			t = Variant::NIL;
+			t = Variant::REAL;
 		} break;
 		case MATH_SIGN: {
-			t = Variant::NIL;
+			t = Variant::INT;
```

Comment:
In GDScript `sign` returns a `float`, so this should likely be `REAL` here too.
https://docs.godotengine.org/en/stable/classes/class_%40gdscript.html#class-gdscript-method-sign

Reply:
Pushed an update with this.